test/verifier/verifier_test.go

```
@@ -179,6 +181,10 @@ func TestVerifier(t *testing.T) {
 				})
 				var ve *ebpf.VerifierError
 				if errors.As(err, &ve) {
+					// Rename object file to avoid subsequent runs to overwrite it,
+					// so we can keep it for CI's artifact upload.
+					os.Rename(objFile, path.Join(*ciliumBasePath, "bpf", name+".o"))
```

Ignored error, maybe log it at least for better visibility?